data/hai/unfettered 0 prologue.txt

```diff
@@ -375,14 +375,14 @@ mission "Unfettered Jump Drive Trading"
 					goto money
 			action
 				outfit "Tripulse Shredder" 2
-			`	They order their crew to do the transfer, and come address you before leaving. "Do note that if you ever want more weapons we will always be willing to trade them for jump drives in the job board. We can of course always offer money instead if that is of higher value to you. Of course, we can only do this exchange here, on <origin>."`
+			`	They order their crew to do the transfer, and come address you before leaving. "Do note that if you ever want more weapons, we will always be willing to trade them for jump drives in the job board. However, we can always offer money instead if that is of higher value to you. Of course, we can only do this exchange here, on <origin>, given it is where these advanced weapons are produced and stocked."`
 				accept
 
 			label dodge
 			`	You manage to dodge the blade but are then left defenseless when the same Unfettered that threw the blade at you comes in with another one, and slashes down to strike. They stop at the last possible moment, with a discontent face. "So this is how you 'fight', Captain? You hide? You dodge? A weapon thrown at you no longer belongs to the enemy, you can claim it!`
```

Comment:
Do we say `captain` or `Captain`?

Reply:
I have no idea but here she wouldnt respect you enough to say captain for sure

Reply:
I feel like it makes more sense for it to be captain so I made all of them be that